cmd/rancherd/auth/auth.go

```diff
@@ -32,18 +34,41 @@ var (
 )
 
 func ResetAdmin(ctx *cli.Context) error {
+	if err := validation(ctx); err != nil {
+		return err
+	}
 	if err := resetAdmin(ctx); err != nil {
 		return errors.Wrap(err, "cluster and rancher are not ready. Please try later.")
 	}
 	return nil
 }
 
-func resetAdmin(_ *cli.Context) error {
+func validation(ctx *cli.Context) error {
+	if ctx.String("password") != "" && ctx.String("password-file") != "" {
+		return errors.New("only one option can be set for password and password-file")
+	}
+	return nil
+}
+
+func resetAdmin(c *cli.Context) error {
```

Comment:
Not blocking the progression of this PR but this might be a good time to update the code to use `clx` as the argument rather than `ctx` to maintain the convention of context and not have the need to make one off argument variable names.